Assets/com.vladfaust.unitywakatime/Editor/Plugin.cs

```diff
@@ -131,7 +133,8 @@ struct Heartbeat {
         : string.Empty;
 
       var heartbeat = new Heartbeat(file, fromSave);
-      if ((heartbeat.time - _lastHeartbeat.time < HEARTBEAT_COOLDOWN) && !fromSave &&
+      if ((heartbeat.time - _lastHeartbeat.time < HEARTBEAT_COOLDOWN) && !fromSave && 
+        //BUG: _lastHeartbeat has no time, entity or type
```

Comment:
It's a good practice to not intervent long conditionals (even multi-line) with comments, put it on top instead.